sematic/resolvers/state_machine_resolver.py

```diff
@@ -230,13 +248,16 @@ def _schedule_future_if_args_resolved(self, future: AbstractFuture) -> None:
             self._execute_future(future)
 
     def _execute_future(self, future: AbstractFuture) -> None:
-        self._future_will_schedule(future)
         if future.props.inline:
+            self._future_will_schedule(future)
             logger.info("Running inline {}".format(future.calculator))
             self._run_inline(future)
-        else:
+        elif self._can_schedule_future(future):
+            self._future_will_schedule(future)
             logger.info("Scheduling {}".format(future.calculator))
             self._schedule_future(future)
+        else:
+            logger.info("Could not schedule {}".format(future.calculator))
```

Comment:
Not quite "could not", this message makes it look like something went wrong, when in reality we are just waiting for resources to open up.

Reply:
Reworded to `"Currently not scheduling {}"`.